src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -719,14 +720,20 @@ public void updateSetting( final IConfigManager manager, final Enum settingName,
 
 	}
 
-	public boolean openGui( final EntityPlayer p, final ICellHandler ch, final ItemStack cell, final EnumFacing side )
+	public boolean openGui( final EntityPlayer p, final EnumFacing side )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "side". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)